doc/programming_guide/asrc/overview.rst

```diff
@@ -149,13 +149,14 @@ To run the app, either xrun or xflash can be used. Connect the XK-VOICE-L71 boar
 
    $ xrun example_asrc_demo.xe
 
-or
+Optionally, xrun ``--xscope`` can be used to provide real-time debug output.
```

Comment:
I think it is better to replace 
xrun ``--xscope`
with 
the command ``xrun --xscope``